pkg/kubelet/kubelet.go

```diff
@@ -204,6 +204,9 @@ const (
 
 	// instrumentationScope is the name of OpenTelemetry instrumentation scope
 	instrumentationScope = "k8s.io/kubernetes/pkg/kubelet"
+
+	// ErrorInconsistentCPUAllocation represent the type of an inconsistentCPUAllocationError
+	ErrorInconsistentCPUAllocation = "inconsistentCPUAllocationError"
```

Comment:
Does this need defining at both kubelet.go and policy_static.go?

Reply:
Thanks, good point. Moved to `kubelet/types/constants.go` constant, with latest commit.